Dockerfile

```diff
@@ -1,29 +1,16 @@
 # This is used to build bindings for arm64
 
 ARG NODE_VERSION
-FROM node:${NODE_VERSION}
+FROM matthewkeil/blst-ts-armbuild:${NODE_VERSION}
```

Comment:
where does this base image come from, can we make it inline here?

Reply:
The inlining is what I got away from because the build takes a really long time (like 30 min) and does not change between runs.  Only changes if the node version changes.  `matthewkeil/blst-ts-armbuild` is built via `Dockerfile.armEnv`.

I added a comment to this file showing where the containers come from and another to `Dockerfile.armEnv` on how to build and publish updated containers.

These both will go away soon (likely before we need node 22) because github is in beta with linux-arm64 runners.  Guessing they will be GA by time we need to add node 22